cpp/modmesh/python/wrapper/modmesh/wrap_SimpleArray.cpp

```diff
@@ -149,7 +149,9 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArray
             .def(
                 "__setitem__",
                 [](wrapped_type & self, const py::ellipsis &, pybind11::array & arr_in)
```

Comment:
(Missed in previous review.)  Other places in the file use `T const &` but here you write `const T &`.  It's OK, but being consistent with the rest in the file is better.